src/fights.ts

```diff
@@ -821,6 +835,45 @@ const freeFightSources = [
     }
   ),
 
+  new FreeFight(
+    () =>
+      have($item`[glitch season reward name]`) &&
+      have($item`unwrapped knock-off retro superhero cape`) &&
+      !get("_glitchMonsterFights") &&
+      get("garbo_fightGlitch", false) &&
+      Array.from(stunDurations).reduce(
+        (sum, [thing, duration]) => sum + (have(thing) ? undelay(duration) : 0),
+        0
+      ) >= 5,
```

Comment:
```suggestion
     sum([...stunDurations], ([thing, duration]) => have(thing) ? undelay(duration) : 0) >= 5
```
Does this cover enough lines? more news as this develops